docs/UserGuide.md

```diff
@@ -250,14 +259,24 @@ Run the <code>clear</code> command to start working with TutorBuddy in a clean s
 
 Format: `clear`
 
+Example:
+1. Type in the command `clear` and press <kbd>Enter</kbd>.
+2. All data in TutorBuddy will be wiped clean.
+![clear_command](images/clear_command.png)
```

Comment:
For this, i'm not sure if it shows much hahaha. Would it be better to show a before and after?

Reply:
kinda agree with you. maybe i should just remove it?

Reply:
actually yes, let's just remove it? haha